src/transformers/modeling_t5.py

```diff
@@ -1027,6 +1027,16 @@ def forward(
 
 @add_start_docstrings("""T5 Model with a `language modeling` head on top. """, T5_START_DOCSTRING)
 class T5ForConditionalGeneration(T5PreTrainedModel):
+    keys_to_ignore_at_load = [
+        r"encoder\.block\.\d+\.layer\.\d+\.layer_norm\.bias",
+        r"encoder\.final_layer_norm\.bias",
```

Comment:
As far as I can tell, layer norm bias is not a thing in t5.
How do you get this warning?